packages/round-manager/src/features/program/ListProgramPage.tsx

```diff
@@ -40,7 +40,7 @@ const ProgramCard: React.FC<ProgramCardProps> = (props: ProgramCardProps) => (
       </CardDescription>
     </CardContent>
     <CardFooter>
-      <CardFooterContent className="justify-end p-6">
+      <CardFooterContent className="justify-end p-6 bg-[#F3F3F5]">
```

Comment:
it's the white area behind the card that should be bg-grey-150 (#F3F3F5) not the card footer.

could you also make this change?

we could also get on a call to explain this

Reply:
Did you mean this area?
<img width="586" alt="fixed" src="https://user-images.githubusercontent.com/28866303/188651920-86d574e9-d23d-45b7-816b-128f8465474a.png">

Reply:
exactly! that area

Reply:
I'm free to hop on call to clear up my confusion also